ansible/roles/biocache-cli/vars/main.yml

```diff
@@ -1,5 +1,5 @@
 # remote assets
-version: "{{ biocache_cli_version | default('1.9.2') }}"
+version: "{{ biocache_cli_version | default('1.9') }}"
```

Comment:
Could you backtrack these default changes as the baseline at ALA has already moved to these versions? You can override them in your local playbooks instead of changing the versions here.